relative-json-pointer.xml

```diff
@@ -91,8 +91,9 @@
                 </preamble>
                 <artwork>
 <![CDATA[
-   relative-json-pointer =  non-negative-integer <json-pointer>
+   relative-json-pointer =  non-negative-integer [index-manipulation] <json-pointer>
    relative-json-pointer =/ non-negative-integer "#"
+   index-manipulation    =  ("+" / "-") non-negative-integer
```

Comment:
Note that this definition allows pointers such as `0-0/foo` (which would be logically equivalent to `0/foo`). I don't feel too strongly one way or another about whether that should be allowed. I'm happy to change it.